Doc/library/venv.rst

```diff
@@ -345,8 +345,8 @@ creation according to their needs, the :class:`EnvBuilder` class.
     .. method:: install_scripts(context, path)
 
         *path* is the path to a directory that should contain subdirectories
-        "common", "posix", "nt", each containing scripts destined for the bin
-        directory in the environment.  The contents of "common" and the
+        ``common``, ``posix``, ``nt``, each containing scripts destined for the
+        bin directory in the environment.  The contents of ``common`` and the
```

Comment:
```suggestion
        ``common``, ``posix``, ``nt``; each containing scripts destined for the
        ``bin`` directory in the environment.  The contents of ``common`` and the
```

* Use literals for ``bin`` as well, since it is a directory like the others
* Use semicolon to more clearly separate the items in the list from the next clause, especially when eliding "and"